site/content/docs/5.2/components/scrollspy.md

```diff
@@ -25,7 +25,7 @@ Scrollspy toggles the `.active` class on anchor (`<a>`) elements when the elemen
 Scroll the area below the navbar and watch the active class change. Open the dropdown menu and watch the dropdown items be highlighted as well.
 
 <div class="bd-example">
-  <nav id="navbar-example2" class="navbar bg-light px-3 mb-3">
+  <nav id="navbar-example2" class="navbar bg-body-secondary px-3 mb-3 rounded-2">
```

Comment:
To be consistent with [other Navbar examples](https://deploy-preview-35857--twbs-bootstrap.netlify.app/docs/5.2/components/navbar/#supported-content)

```suggestion
  <nav id="navbar-example2" class="navbar bg-body-tertiary px-3 mb-3 rounded-2">
```